cmd/podman/create_cli.go

```diff
@@ -121,20 +139,20 @@ func validateVolumeOpts(option string) error {
 	for _, opt := range options {
 		switch opt {
 		case "rw", "ro":
+			foundRWRO++
 			if foundRWRO > 1 {
 				return errors.Errorf("invalid options %q, can only specify 1 'rw' or 'ro' option", option)
 			}
-			foundRWRO++
 		case "z", "Z":
+			foundLabelChange++
 			if foundLabelChange > 1 {
 				return errors.Errorf("invalid options %q, can only specify 1 'z' or 'Z' option", option)
 			}
-			foundLabelChange++
 		case "private", "rprivate", "shared", "rshared", "slave", "rslave":
+			foundRootPropagation++
```

Comment:
these options aren't listed in the man page, should they be?

Reply:
Same as above.